.github/workflows/builds.yml

```diff
@@ -11,46 +11,65 @@ env:
   CTEST_OUTPUT_ON_FAILURE: 1
 
 jobs:
-  ubuntu2004:
+  linux:
     runs-on: ubuntu-latest
-    container: gitlab-registry.cern.ch/acts/machines/ubuntu2004:v4
+    container: ghcr.io/acts-project/${{ matrix.image }}:v5
+    strategy:
+      matrix:
+        image:
+          - centos7-lcg95apython3
+          - centos7-lcg96
+          - ubuntu2004
+    env:
+      SETUP: true
+      INSTALL_DIR: ${{ github.workspace }}/install
     steps:
       - uses: actions/checkout@v2
+      - name: Define setup script
+        if: contains(matrix.image, 'lcg')
+        run: echo "::set-env name=SETUP::source /opt/lcg_view/setup.sh"
       - name: Configure
+        # use manual mkdir+cd. CMake on LCG95 does not like -B ... -S ...
         run: >
-          cmake -B build -S .
+          ${SETUP} &&
+          mkdir build && cd build &&
+          cmake ..
           -GNinja
           -DCMAKE_BUILD_TYPE=Release
           -DCMAKE_CXX_FLAGS=-Werror
+          -DCMAKE_INSTALL_PREFIX="${INSTALL_DIR}"
           -DACTS_BUILD_EVERYTHING=on
       - name: Build
-        run: cmake --build build --
+        run: ${SETUP} && cmake --build build -- 
       - name: Unit tests
-        run: cmake --build build -- test
+        run: ${SETUP} && cmake --build build -- test
       - name: Integration tests
-        run: cmake --build build -- integrationtests
-  cuda:
-    runs-on: ubuntu-latest
-    container: gitlab-registry.cern.ch/acts/machines/ubuntu1804_cuda:v4
-    steps:
-      - uses: actions/checkout@v2
-      - name: Configure
+        run: ${SETUP} && cmake --build build -- integrationtests
+      - name: Install
+        run: ${SETUP} && cmake --build build -- install
+      - uses: actions/upload-artifact@v2
+        with:
+          name: acts-${{ matrix.image }}
+          path: ${{ env.INSTALL_DIR }}
```

Comment:
Do we need this?

Reply:
I needed it to debug issues with the resulting installed files. It could be helpful to others in the future or if somebody just wants to use a compiled version.

Reply:
Our output can grow quite large. I'd prefer to not use the artifacts feature if we don't need to, tbh.

Reply:
But ok, if you feel there's a benefit.